apps/settings/js/settings.js

```diff
@@ -97,12 +97,6 @@ window.Settings = {
         'tabletAndLandscaped',
         '(min-width: 768px) and (orientation: landscape)');
       window.addEventListener('screenlayoutchange', this.rotate);
-
-      // WifiHelper is guaranteed to be loaded in main.js before calling to
-      // this line.
-      if (this.isTabletAndLandscape()) {
-        this.currentPanel = this.initialPanelForTablet;
-      }
```

Comment:
Duh!

Could you also remove other use of this initialPanelForTablet? I've seen some other cases where it is used (like when rotating screens: we may support that).